src/contention_detectors/overload.hpp

```diff
@@ -48,23 +32,29 @@ class OverloadDetector :
   OverloadDetector(
       Consumer<Contentions>* _consumer,
       const lambda::function<usage::GetterFunction>& _cpuUsageGetFunction,
-      SerenityConfig _conf,
+      const SerenityConfig& _conf,
       const Tag& _tag = Tag(QOS_CONTROLLER, NAME))
     : tag(_tag),
       cpuUsageGetFunction(_cpuUsageGetFunction),
       Producer<Contentions>(_consumer) {
-    SerenityConfig config = OverloadDetectorConfig(_conf);
-    this->cfgUtilizationThreshold =
-      config.getD(detector::THRESHOLD);
+    // Parse config values.
+    setCfgUtilizationThreshold(
+      _conf.item<double_t>(detector::THRESHOLD,
```

Comment:
`detector::THRESHOLD` should be named `UTILIZATION_THRESHOLD_KEY` and declared as static constexpr in OverloadDetector. Similar for `DEFAULT_UTILIZATION_THRESHOLD`.